vmprof generates malfunction output with 64 bit Python in Windows

Synopsis:

```
>python -m vmprof -o prof-x64.log sometest.py

>python -m vmprof.upload prof-x64.log
Traceback (most recent call last):
  File "C:\Python27_x64\lib\runpy.py", line 174, in _run_module_as_main
    "__main__", fname, loader, pkg_name)
  File "C:\Python27_x64\lib\runpy.py", line 72, in _run_code
    exec code in run_globals
  File "C:\Python27_x64\lib\site-packages\vmprof\upload.py", line 105, in <module>
    main()
  File "C:\Python27_x64\lib\site-packages\vmprof\upload.py", line 94, in main
    stats = vmprof.read_profile(args.profile)
  File "C:\Python27_x64\lib\site-packages\vmprof\profiler.py", line 34, in read_profile
    period, profiles, virtual_symbols, interp_name = read_prof(prof_file)
  File "C:\Python27_x64\lib\site-packages\vmprof\reader.py", line 172, in read_prof
    assert read_word(fileobj) == 0 # header count
  File "C:\Python27_x64\lib\site-packages\vmprof\binary.py", line 19, in read_word
    r = int(struct.unpack('l', b)[0])
struct.error: unpack requires a string argument of length 4

>python -V
Python 2.7.12

>python -c "import sys; print(sys.version)"
2.7.12 (v2.7.12:d33e0cf91556, Jun 27 2016, 15:24:40) [MSC v.1500 64 bit (AMD64)]
```

It works with x86 Python:

```
>python -m vmprof -o prof-x86.log sometest.py

>python -m vmprof.upload prof-x86.log
Compiling and uploading to http://vmprof.com...
VMProf log: http://vmprof.com/#/e6881666e8da424f33b18ecfd767d708

C:\Working\Repositories>python -V
Python 2.7.12

C:\Working\Repositories>python -c "import sys; print(sys.version)"
2.7.12 (v2.7.12:d33e0cf91556, Jun 27 2016, 15:19:22) [MSC v.1500 32 bit (Intel)]
```

```
>python -m vmprof.upload prof-x64.log
Traceback (most recent call last):
  File "C:\Python27_x86\lib\runpy.py", line 174, in _run_module_as_main
    "__main__", fname, loader, pkg_name)
  File "C:\Python27_x86\lib\runpy.py", line 72, in _run_code
    exec code in run_globals
  File "C:\Python27_x86\lib\site-packages\vmprof\upload.py", line 105, in <module>
    main()
  File "C:\Python27_x86\lib\site-packages\vmprof\upload.py", line 94, in main
    stats = vmprof.read_profile(args.profile)
  File "C:\Python27_x86\lib\site-packages\vmprof\profiler.py", line 34, in read_profile
    period, profiles, virtual_symbols, interp_name = read_prof(prof_file)
  File "C:\Python27_x86\lib\site-packages\vmprof\reader.py", line 231, in read_prof
    assert not marker, (fileobj.tell(), repr(marker))
AssertionError: (69L, "'0'")
```

The content of `sometest.py`:

``` python
import time

for _ in xrange(1000):
    time.sleep(.001)
```

Feel free to ask for additional information which can help to resolve this issue. Thanks!

--- a/setup.py
+++ b/setup.py
@@ -33,6 +33,7 @@ setup(
     url='https://github.com/vmprof/vmprof-python',
     install_requires=[
         'requests',
+        'six',
     ],
     entry_points = {
         'console_scripts': [

